config.go

```diff
@@ -57,5 +59,6 @@ func NewDefaultConfig() Config {
 		EventsPort:      10001,
 		EventsInterface: "eth0",
 		LogOutput:       ioutil.Discard,
+		PollingWaitTime: 500,
```

Comment:
If we retype `PollingWaitTime`, we should do `500 * time.Millisecond` here. (I know that milliseconds is default, but typing it out (pun intended) makes things more clear.)